contrib/packs/actions/create.meta.yaml

```diff
@@ -0,0 +1,41 @@
+---
+  name: "create"
+  runner_type: "action-chain"
+  description: "Creates a new pack"
+  enabled: true
+  entry_point: "workflows/create.yaml"
+  parameters:
+    name:
+      type: "string"
+      required: true
+      description: "Name of the pack to create"
+    description:
```

Comment:
I feel like the description should be required at this point: we'll have the index, and the front-end, and the description is prominent in both, so the packs without description will suck. Wdyt?